cves/CVE-2018-13304.yml

```diff
@@ -154,10 +153,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: There are unit tests for all three changed files
```

Comment:
I'd elaborate and mention if these areas/error-causing paths were tested. Many times coverage is lacking.